### DISCUSSION TITLE
How to change netbeans 13 menu bar background style to origin 12.6 version

### DISCUSSION THREAD

**Discussion by rfrkk** [apache/netbeans#3895] [General]
**How to change netbeans 13 menu bar background style to origin 12.6 version ?**

![Snipaste_2022-03-30_12-50-22](https://user-images.githubusercontent.com/137764/160756158-02f961e7-bc24-4a48-9bdb-44e75080c919.png)

**Comment by geertjanw** [comment]
As always, you'll go to the Options window under the Tools menu to configure the NetBeans appearance: Tools | Options | Appearance | Look and Feel.

**Reply by rfrkk** [reply]
Thank you for solving my problem
